rabies/parser.py

```diff
@@ -169,8 +169,8 @@ def get_parser():
             "\n"
         )
     preprocess.add_argument(
-        "--HMC_option", type=str, default='intraSubjectBOLD',
-        choices=['intraSubjectBOLD', '0', '1', '2', '3'],
+        "--HMC_option", type=str, default='intraSubjectBOLDLatest',
```

Comment:
Changed the default call to be the latest ANTs toolkit version call as briefly discussed before with @Gab-D-G.